Sources/Jobs/JobQueueDriver.swift

```diff
@@ -88,8 +88,10 @@ public struct JobQueueResult<JobID: Sendable>: Sendable {
 
 /// Protocol for JobOptions
 public protocol JobOptionsProtocol: Sendable {
-    /// Initialize job options with default value
-    init()
+    /// When to execute the job
+    var delayUntil: Date { get }
+
+    init(delayUntil: Date)
```

Comment:
Alternative idea, what about we keep both? Right now there _is_ no delay until right? So this would be equivalent

```swift
init()
init(delayUntil: Date)

extension JobOptionsProtocol {
  var delayUntil: Date { .now }
  init(delayUntil date: Date) { self.init() }
}
```